src/data/matrix/basic.lean

```diff
@@ -818,27 +820,27 @@ by { ext i j, refl }
   (- M)ᵀ = - Mᵀ  :=
 by ext i j; refl
 
-lemma transpose_map {β : Type w} {f : α → β} {M : matrix m n α} : Mᵀ.map f = (M.map f)ᵀ :=
+lemma transpose_map {f : α → β} {M : matrix m n α} : Mᵀ.map f = (M.map f)ᵀ :=
 by { ext, refl }
 
 end transpose
 
 section star_ring
-variables [decidable_eq n] {R : Type*} [semiring R] [star_ring R]
+variables [decidable_eq n] [semiring α] [star_ring α]
```

Comment:
Hmm, now you are moving from "meaningful variable name" to "generic type variable name". Why?

Reply:
I think we should rather change the rest of the file. But in another PR.

Reply:
I agree, although I'm not sure what letter I would use for the scalar action type if `R` were reserved for the coefficients.